GitUI/TranslatedStrings.cs

```diff
@@ -312,6 +312,17 @@ public static void Reinitialize()
         private readonly TranslationString _scriptErrorOptionWithoutRevisionText = new("A valid revision is required to substitute the argument options");
         private readonly TranslationString _scriptText = new("Script");
 
+        private readonly TranslationString _seeErrorMessage = new("See error message");
+        private readonly TranslationString _hideErrorMessage = new("Hide error message");
```

Comment:
For `_gitDubiousOwnershipXxx` we add "..." at the end
```suggestion
        private readonly TranslationString _seeErrorMessage = new("See error message...");
        private readonly TranslationString _hideErrorMessage = new("Hide error message...");
```